client/my-sites/plans/trials/trial-acknowledge/with-onclick-trial-request.tsx

```diff
@@ -24,29 +14,10 @@ export const WithOnclickTrialRequest = createHigherOrderComponent(
 		const dispatch = useDispatch();
 		const siteId = useSelector( getSelectedSiteId ) as number;
 		const locale = useSelector( getCurrentUserLocale );
-		const transferStatus = useSelector( ( state ) => getAutomatedTransferStatus( state, siteId ) );
-		const isSiteAtomic = useSelector( ( state ) => isSiteWpcomAtomic( state, siteId as number ) );
-		const isFetchingTransferStatus = useSelector( ( state ) =>
-			isFetchingAutomatedTransferStatus( state, siteId )
-		);
-
-		useEffect( () => {
-			if ( siteId && isSiteAtomic ) {
-				return;
-			}
-			if ( ! isFetchingTransferStatus && transferStatus !== transferStates.COMPLETE ) {
-				waitFor( 2 ).then( () => dispatch( fetchAutomatedTransferStatus( siteId ) ) );
-			}
-			// Once the transferStatus is reported complete, query the sites endpoint
-			// until the `is_wpcom_atomic` = true is returned
-			if ( transferStatus === transferStates.COMPLETE && ! isSiteAtomic ) {
-				waitFor( 2 ).then( () => dispatch( requestSite( siteId ) ) );
-			}
-		}, [ transferStatus ] );
 
 		const fetchUpdatedData = () => {
 			//after transfer is complete we wait to fetch update site with `is_wpcom_atomic`
-			waitFor( 2 ).then( () => dispatch( requestSite( siteId ) ) );
+			waitFor( 3 ).then( () => dispatch( requestSite( siteId ) ) );
```

Comment:
Would it make sense to poll this information instead of waiting an arbitrary number of seconds?

Reply:
Yup that's the intention!

Reply:
We poll in the `<HostingActivateStatus>` component. but we don't do any dispatch there, keeping the relevent functions to be dispatched out of that component, the component only returns the status.